skeleton/docker-entrypoint.sh

```diff
@@ -28,6 +28,8 @@ fi
 [ -z ${COMPILE_MO_FILES+x} ] && export COMPILE_MO_FILES=true
 [ -z ${DEBUG_MODE+x} ] && export DEBUG_MODE=off
 [ -z ${ZOPE_FORM_MEMORY_LIMIT+x} ] && export ZOPE_FORM_MEMORY_LIMIT=4MB
+[ -z ${ZOPE_FORM_DISK_LIMIT+x} ] && export ZOPE_FORM_DISK_LIMIT=4MB
+[ -z ${ZOPE_FORM_MEMFILE_LIMIT+x} ] && export ZOPE_FORM_MEMFILE_LIMIT=4MB
```

Comment:
Let's keep the same defaults as in Zope please, so that it doesn't suddenly change when someone updates the image.
```suggestion
[ -z ${ZOPE_FORM_DISK_LIMIT+x} ] && export ZOPE_FORM_DISK_LIMIT=1GB
[ -z ${ZOPE_FORM_MEMFILE_LIMIT+x} ] && export ZOPE_FORM_MEMFILE_LIMIT=4KB
```